grpc/src/main/java/io/stargate/grpc/service/ExceptionHandler.java

```diff
@@ -134,6 +163,7 @@ private void handlePersistenceException(PersistenceException pe) {
   }
 
   private void handleUnavailable(UnavailableException ue) {
+    LOG.error(getErrorMessage(Status.DEADLINE_EXCEEDED, ue));
```

Comment:
logged status is not the same as the one reported back?

Reply:
Is there a better `Status` code to use?